app/code/Magento/Customer/Model/Options.php

```diff
@@ -47,10 +47,11 @@ public function __construct(
      */
     public function getNamePrefixOptions($store = null)
     {
-        return $this->prepareNamePrefixSuffixOptions(
-            $this->addressHelper->getConfig('prefix_options', $store),
-            $this->addressHelper->getConfig('prefix_show', $store) == NooptreqSource::VALUE_OPTIONAL
-        );
+        $options = $this->addressHelper->getConfig('prefix_options', $store);
```

Comment:
please move logic to `prepareNamePrefixSuffixOptions()`

Reply:
Hello @xmav 
Thank you for your review
Fixed, please check.